satpy/tests/reader_tests/test_avhrr_l1b_gaclac.py

```diff
@@ -194,13 +195,62 @@ def test_get_dataset_channels(self, get_channel, *mocks):
             fh.get_dataset(key=key, info={'name': 1})
             get_channel.assert_called_with(key)
 
+    @mock.patch('satpy.readers.avhrr_l1b_gaclac.GACLACFile.__init__', return_value=None)
+    @mock.patch('satpy.readers.avhrr_l1b_gaclac.GACLACFile.read_raw_data')
+    @mock.patch('satpy.readers.avhrr_l1b_gaclac.GACLACFile._get_channel')
+    def test_get_dataset_no_tle(self, get_channel, *mocks):
```

Comment:
Would it make sense to add a test for `GACLACFile._update_attrs` instead of repeating the test for `GACLACFile.get_dataset` here?

Reply:
I'd rather avoid testing private methods directly, because it will lock us into a given implementation. Rather, I prefer to test only the public interfaces. But I will try to remove some code duplication here.

Reply:
Ok, that's a good point!

Reply:
@sfinkens what do you think of this now?